examples/research_projects/onnxruntime/unconditional_image_generation/train_unconditional.py

```diff
@@ -105,6 +97,21 @@ def parse_args():
             " resolution"
         ),
     )
+    parser.add_argument(
+        "--center_crop",
+        default=False,
+        action="store_true",
```

Comment:
Both --center_crop and --random_flip defaulted to `True` before the changes no? Could we make sure we still have the same default values to not break the scripts?